src/main/java/com/spartronics4915/frc2020/subsystems/Climber.java

```diff
@@ -1,11 +1,19 @@
 package com.spartronics4915.frc2020.subsystems;
 
 import com.spartronics4915.lib.subsystems.SpartronicsSubsystem;
+import com.spartronics4915.lib.hardware.motors.SpartronicsSRX;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
 
 public class Climber extends SpartronicsSubsystem {
 
+    private TalonSRX mClimber775PRO;
+    private CANSparkMax mClimberNEO;
     public Climber() {
         //Hardware Contructor (Add motors and such here when I get them)
+        mClimber775PRO = new TalonSRX(5);
```

Comment:
You need to move these motor IDs to `Constants.java` and call them from there. 

ex. `public static final int mClimber775PROID`